styles/styles.css

```diff
@@ -458,6 +458,13 @@ main .section.highlight {
   overflow-wrap: break-word;
 }
 
+.block > div > [data-align$="right"] {
```

Comment:
padding, margin and overflow-wrap seem without effect. remove?

Reply:
@solaris007 if you mean the positioning is off because of the extra space this is the normal behaviour of columns, it adds bottom margin between each cell, I can't change this or I'll break all the other positioning in all the columns used. That or I create a custom block just for this case scenario. And IMO this looks better

Reply:
i am saying when disabling these properties in dev tools, i see no effect, therefore it smells like dead weight.

Reply:
That I got, I was talking about the positioning, I will delete the extra bits and resubmit